appengine/standard/django/requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyMySQL==0.9.3
-Django<2.0.0,>=1.11.8
+Django==1.11.28 # needs to stay under 2.0.0
```

Comment:
nit: can we add a reason? So future us know why we pinned?

Reply:
done - good nit